src/oss-risk-calculator/RiskCalculatorTool.cs

```diff
@@ -53,14 +51,15 @@ static void Main(string[] args)
 
             if (((IList<string>)riskCalculator.Options["target"]).Count > 0)
             {
-                Dictionary<string, int> freq = new Dictionary<string, int>();
+                string destinationDirectory = (string)riskCalculator.Options["cache-directory"];
+                bool doCaching = string.IsNullOrEmpty(destinationDirectory);
```

Comment:
This looks like the opposite logic of what you normally do where caching is set based on destinationDirectory being not null.

Reply:
I should stop late night coding